Base second edition

generalpepper opened the pull request.

Commit:
Artisan, Bandit

Commit:
Harbinger, Merchant

Commit:
Poacher, Sentry

srmcconomy commented on src/server/models/Player.js:
```diff
@@ -101,14 +101,16 @@ export default class Player extends Model {
   async trash(card, from = 'hand') {
     this.emit('before-trash', card);
-    this.moveCard(card, this[from], this.game.trash);
+    let whereFrom = typeof from === 'string' ? await this[from] : from;
```
should be `const` if never redefined
`await` doesnt do anything here

Commit:
Fix sentry

generalpepper commented:
Fixed trash and discard functions.

Fixed Sentry so that it will call onTrash() and onDiscard() methods

Have not Done Library yet, haven't decided cleanest way to move a card from aside to hand. Either:
1) player.pickUp(card,aside); and change pickUp to call onDraw() if 'from' is not playArea

2) player.draw(card, 'hand', aside); and introduce a from input to draw()

Thoughts?
